packages/devtools_app/release_notes/helpers/release_notes_template.md

```diff
@@ -1,6 +1,17 @@
 This is draft for future release notes, that are going to land on
 [the Flutter website](https://docs.flutter.dev/tools/devtools/release-notes).
 
+Template for a release notes item:
+
+------------
+* Enabled export of snapshots and improved snapshotting performance. -
+[#7197](https://github.com/flutter/devtools/pull/7197),
+[#7439](https://github.com/flutter/devtools/pull/7439),
+[#7449](https://github.com/flutter/devtools/pull/7449)
+
+    ![Export snapshot](/tools/devtools/release-notes/images-2.34.1/7197-export.png "Export snapshot")
+------------
+
```

Comment:
why are we adding this to the template? this will get copied over to NEXT_RELEASE_NOTES.md each time we have a new set of release notes, which we don't wan't.

Reply:
the release notes readme might be a better place for an example like this. It already has an example for adding an image.